src/ripple/rpc/handlers/AccountCurrenciesHandler.cpp

```diff
@@ -46,19 +46,19 @@ doAccountCurrencies(RPC::JsonContext& context)
         params.isMember(jss::account) ? params[jss::account].asString()
                                       : params[jss::ident].asString());
 
-    bool const bStrict =
-        params.isMember(jss::strict) && params[jss::strict].asBool();
-
     // Get info on account.
-    AccountID accountID;  // out param
-    if (auto jvAccepted = RPC::accountFromString(accountID, strIdent, bStrict))
-        return jvAccepted;
+    auto const accountID = parseBase58<AccountID>(strIdent);
```

Comment:
`AccountID` is not really optional. Please consider the updated suggested in `AccountChannels.cpp` comment. I will not repeat this comment for other files. All changed files under `rpc/handlers` have a similar code block: AccountInfo.cpp, AccountLines.cpp, AccountObjects.cpp, AccountOffers.cpp, DepositAuthorized.cpp, GatewayBalances.cpp, NoRipplecheck.cpp, OwnerInfo.cpp.

Reply:
I have applied your suggested changes across the board. Let me know if you find any further issues.